integration_test.go

```diff
@@ -120,7 +120,8 @@ func testHelperCartographer(
 	defer termFunc()
 
 	attrCfg := &vcConfig.Config{
-		Camera: map[string]string{"name": "stub_lidar"},
+		Camera:         map[string]string{"name": "stub_lidar"},
+		MovementSensor: map[string]string{"name": "stub_imu"},
```

Comment:
[q] How does/will the stub imu values affect the integration test results?

Reply:
I don't believe they do, it's more a matter of can we retrieve the data, is my understanding. But to be fair, I don't fully understand the point of the stub_lidar / stub_imu and no one has been able to do a great job of explaining it to me so far.